src/engine/client/glshader.cpp

```diff
@@ -77,9 +91,15 @@ GLuint LoadShader(const char *vertex_path, const char *fragment_path) {
 
     glGetProgramiv(program, GL_LINK_STATUS, &result);
     glGetProgramiv(program, GL_INFO_LOG_LENGTH, &logLength);
-    //std::vector<char> programError( (logLength > 1) ? logLength : 1 );
-    //glGetProgramInfoLog(program, logLength, NULL, &programError[0]);
-    //std::cout << &programError[0] << std::endl;
+    if (!result)
+    {
+        dbg_msg("gfx", "Could not link shaders %s %s", vertex_path, fragment_path);
+        char *infoLog = (char *) malloc ( sizeof ( char ) * logLength );
+        glGetShaderInfoLog(program, logLength, NULL, infoLog);
+        dbg_msg("gfx", "%s", infoLog);
+        free ( infoLog );
+        return 0;
+    }
 
     glDeleteShader(vertShader);
```

Comment:
The game will run. Just try it :P You just won't see any shaders, but tbh it doesn't even look too too bad (I though it'd be horrible)